projects/ion/src/lib/modal/component/modal.component.ts

```diff
@@ -36,8 +36,10 @@ export class IonModalComponent implements OnInit {
   ionOnClose = new EventEmitter<IonModalResponse | undefined>();
 
   private componentFactory: ComponentRef<unknown>;
+  public DEFAULT_WIDTH = 500;
   private _defaultModal: IonModalConfiguration = {
     title: 'Ion Modal',
+    width: this.DEFAULT_WIDTH,
     showOverlay: true,
     overlayCanDismiss: true,
```

Comment:
```suggestion
  private _defaultModal: IonModalConfiguration = {
    title: 'Ion Modal',
    showOverlay: true,
    overlayCanDismiss: true,
```
You're not using this anymore.